lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -206,6 +211,100 @@ def setUp(self):
             ]
         )
 
+    @mock.patch('xmodule.x_module.XModuleMixin.has_dynamic_children', mock.Mock(return_value='True'))
+    def test_view_redirect_if_entrance_exam_required(self):
```

Comment:
Sorry, I meant to say "PASSED" the entrance exam. I have not passed it-- "To access course materials, you must score 80% or higher on this exam. Your current score is 0%."